.github/workflows/format.yml

```diff
@@ -20,4 +20,5 @@ jobs:
       - name: Check project is formatted
         uses: jrouly/scalafmt-native-action@v2
         with:
-          version: '3.0.7'
+          version: '3.7.2'
+          arguments: '--list --mode diff-ref=origin/master'
```

Comment:
This means that scalafmt for the CI will only run on files which have been changed from `origin/master` according to git.

Reply:
👍